proto/sqs/chainregistry/v1beta1/query.proto

```
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package sqs.chainregistry.v1beta1;
+
+import "google/api/annotations.proto";
```

Buf (1.47.2)

import "google/api/annotations.proto": file does not exist

(COMPILE)